packages/toast/src/toast.tsx

```diff
@@ -65,6 +65,7 @@ export const Toast: React.FC<ToastProps> = (props) => {
     message,
     onCloseComplete,
     onRequestRemove,
+    maxWidth = 560, 
```

Comment:
This looks interesting to me. Instead of exposing just `maxWidth`, I would expose `style` so the user can handle any other edge-case unrelated to `maxWidth`.

What do you think?

Reply:
my reason for not doing so was actually really simple - i didn't want to assume that `minWidth`/`margin` should be customizable. due to my limited experience with frontend code, i wasn't sure if there were other considerations beyond [modal styling](https://css-tricks.com/considerations-styling-modal/#dealing-with-width) and wanted to err on the safe side. 

that being said, if you foresee a usecase for allowing customisable `minWidth` and `margin`, i would be more than happy to make the appropriate changes!

Reply:
@segunadebayo just double checking here - should i make the change and expose `style`?

Reply:
Yes please.